.vscode/launch.json

```diff
@@ -17,12 +18,13 @@
       "name": "Test specific packages",
       "runtimeVersion": "8.16.0",
       "runtimeExecutable": "${workspaceFolder}/node_modules/.bin/babel-node",
+      "env": {"LOGGER_LEVEL": "log", "DEBUG": "messages"},
       "runtimeArgs": [
         "./tooling/index.js",
         "test",
         "--packages",
-        "@webex/${input:packages}",
-        "${input:flags}"
+        "@webex/plugin-messages",
```

Comment:
This change should be reverted so that any package can be tested.

Reply:
Done.